brion/target.cpp

```diff
@@ -147,11 +147,12 @@ const Strings& Target::get( const std::string& name ) const
 
 namespace
 {
-void _parse( const Targets& targets, const std::string& name, GIDSet& gids )
+void _parse( const Targets& targets, const std::string& name, GIDSet& gids, bool& found )
```

Comment:
Probably the best solution. Dealing with the return value vs recursion is annoying probably